cli.js

```diff
@@ -543,7 +543,7 @@ process.on('uncaughtException', function(err) {
           return ui.log('warn', 'You must provide an registry name to create.');
         if (!args[3])
           return ui.log('warn', 'You must provide the registry module name to generate from.');
-        return Promise.resolve(registry.create(args[2], args[3]))
+        return Promise.resolve(registry.create(args[2], args[3], undefined, options.local))
```

Comment:
Let's alter the signature rather to `return Promise.resolve(registry.create(args[2], args[3], options.local))`